### PR TITLE
RUN-4368 id undefined issue

### PR DESCRIPTION
**Pull request by licui3936**
This PR audited application APIs and fixed a few undefined issues found when an app doesn't exist. 
Tests: 
[Windows 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b92899a6107b45e6d1ebc40)
[Windows 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b928b016107b45e6d1ebc41)

### CODE REVIEW

**Comment by rdepena** [comment]
@licui3936 What about errors on applications that are not running ?

**Comment by licui3936** [comment]
@rdepena , i tested 'app not running' case,  there is no exception.  the APIs still return the results.

File: src/browser/api/application.js [hunk 1]
```diff
@@ -837,18 +861,27 @@ Application.setTrayIcon = function(identity, iconUrl, callback, errorCallback) {
...
-    // set zoom level for each child window
-    app.children.forEach(function(childWindow) {
-        const childWindowIdentity = {
-            name: childWindow.openfinWindow.name,
-            uuid: childWindow.openfinWindow.uuid
-        };
-        Window.setZoomLevel(childWindowIdentity, level);
-    });
+    if (appError) {
+        errorCallback(new Error(appError));
```
Comment (HarsimranSingh) [inline src/browser/api/application.js]: Need to check `typeof errorCallback === 'function'` just like `callback`

File: src/browser/api/application.js [hunk 2]
```diff
@@ -837,18 +861,27 @@ Application.setTrayIcon = function(identity, iconUrl, callback, errorCallback) {
     });
 };
 
-Application.setZoomLevel = function(identity, level) {
-    const app = coreState.appByUuid(identity.uuid);
+Application.setZoomLevel = function(identity, level, callback, errorCallback) {
```
Comment (HarsimranSingh) [inline src/browser/api/application.js]: https://github.com/HadoukenIO/core/blob/develop/src/browser/api_protocol/api_handlers/application.js#L94 callback and errorcallback aren't even passed in here

File: src/browser/api/application.js (hunk 1 above)
Comment (licui3936) [reply src/browser/api/application.js]: Fixed it. Actually we don't need callback and errorcallback in this function.

File: src/browser/api/application.js (hunk 2 above)
Comment (licui3936) [reply src/browser/api/application.js]: Please see other comments.

**Pull request merged**